src/components/Head.astro

```diff
@@ -30,7 +30,7 @@ const { title, description, type, image } = Astro.props;
       },
     }}
     twitter={{
-      creator: '@d_bozhinovski',
+      creator: '@beerjs_phx',
```

Comment:
should be phx_tech_council? not sure how @monteslu has it formatted on the twitbots

Reply:
I was just going with the twitter account that was talking about us. But I'm cool with phx tech too since they are the parent company.